contents/articles/2017-01_saliency-in-vr/index.md

```diff
@@ -0,0 +1,22 @@
+--- 
+title: "Saliency in VR: How do people explore virtual environment"
+date: Mon Jan 09 2017 11:45:18 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Vincent et al., proposes new methods to learn and predict time-dependent saliency in VR to make eye tracking expendable..."
+featured_img: "../../../../media/images/blog/vr_saliency.png"
+featured_img_thumb: "../../../../media/images/blog/thumb/vr_saliency.png"
+---
+
+<img src="../../../../media/images/blog/vr_saliency.png" class='Feature-image u-padBottom--1' alt="Ground Truth Saliency Map">
+Image Source: [Saliency in VR: How do people explore virtual environments?](https://arxiv.org/pdf/1612.04335.pdf)
+
+<br>
+The experiences offered by virtual environments differ from how we experience the real physical world and other forms of media. View behavior in immersive scenarios are much more complex than conventional displays due to the technology of interactions in kinematics that makes VR possible. 
```

Comment:
In their recent research paper [Saliency in VR: How do people explore virtual environments?], Vincent Sitzmann et al., argue that viewing behavior in VR environments is much more complex than on conventional displays due to the technology of interactions in kinematics that makes VR possible.